packages/dotcom-build-js/src/babel.ts

```diff
@@ -1,46 +1,65 @@
-import { hooks } from './hooks'
 import { PluginOptions } from './types'
-import { ConfigContext } from '@financial-times/dotcom-build-webpack-config'
 
-export default (options: PluginOptions = {}, context: ConfigContext) => {
-  const presetReactOptions = context.publish(hooks.BABEL_PRESET_REACT_OPTIONS, {
+function getBabelConfig(options: PluginOptions = {}) {
+  const presetEnvOpts = {
+    targets: ['last 2 Chrome versions', 'ie 11', 'Safari >= 9.1', 'ff ESR', 'last 2 Edge versions'],
+    // Exclude transforms that make all code slower
+    // See https://github.com/facebook/create-react-app/pull/5278
+    exclude: ['transform-typeof-symbol']
+  }
+
+  const presetReactOptions = {
     pragma: options.jsxPragma,
     pragmaFrag: options.jsxPragmaFrag
-  })
+  }
 
-  const presetTypescriptOptions = context.publish(hooks.BABEL_PRESET_TYPESCRIPT_OPTIONS, {
+  const presetTypescriptOptions = {
     jsxPragma: options.jsxPragma
-  })
-
-  const pluginSyntaxDynamicImportOptions = context.publish(hooks.BABEL_PLUGIN_SYNTAX_DYNAMIC_IMPORT_OPTIONS, {})
-
-  const pluginClassPropertiesOptions = context.publish(hooks.BABEL_PLUGIN_CLASS_PROPERTIES_OPTIONS, {})
+  }
 
-  const pluginTransformRuntimeOptions = context.publish(hooks.BABEL_PLUGIN_TRANSFORM_RUNTIME_OPTIONS, {
+  const pluginTransformRuntimeOptions = {
     // You might think we'd want to abstract the helper functions so they can be reused but doing so
     // means we generate unstable hashes because the generated helper modules are at the bottom of
     // the dependency tree but their contents depends on the features each app uses. Inlining them
     // adds little (usually <1kb) to our total JS payload because the minimizer can usually reduce
     // them down and actually result in fewer function calls overall!
     // <https://github.com/Financial-Times/dotcom-page-kit/issues/576>
     helpers: false
-  })
+  }
 
   const config = {
+    // By default Babel assumes all source code is ESM so force it to check for CJS
+    sourceType: 'unambiguous',
+    babelrc: true,
```

Comment:
Should this be false? The readme says "Other methods of configuring Babel are not supported" does that impy not possible or you're on your own?

Reply:
this hasn't changed from the old config. i'm implying they're on their own